gmp/xml.py

```diff
@@ -57,6 +57,10 @@ def add_element(self, name, text=None, attrs=None):
     def set_attribute(self, name, value):
         self._element.set(name, value)
 
+    def set_attributes_from_args(self, kwargs):
```

Comment:
Would be nice to change it to
```suggestion
    def set_attributes(self, attrs):
        """Set several attributes at once.
    
        Arguments:
            attrs (dict): Attributes to be set on the element
        """
```